### PR TITLE
Re-add the version control script

### LINKED ISSUE #33: Support automatically update versions in Function Mesh docs
image

### PR DESCRIPTION
**Pull request by zeo1995**
Fixed #33 
* Re-add the version control script
* Generate 0.1.4 document
* Add script for auto replace version

### CODE REVIEW

**Review by Huanli-Meng — CHANGES_REQUESTED** [review]
The `docs/overview.md` should not be covered in this PR. And i cannot access to the preview link of the function mesh website.

File: replace.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    //console.error(error)
+    console.error('no versions found defaulting to 2.1.0')
+  }
+  return ['1.1.0']
+}
+
+function doReplace(options) {
+  replace(options)
+    .then(changes => {
+      console.log(changes)
```
Comment (tuteng) [inline replace.js:19]: Delete this debug

**Comment by tuteng** [comment]
ping @Huanli-Meng PTAL

**Review by Huanli-Meng — APPROVED** [review]

**Comment by Huanli-Meng** [comment]
> ping @Huanli-Meng PTAL

approved

**Review by tuteng — APPROVED** [review]

**Pull request merged**